middlewares/error-handlers.js

```diff
@@ -0,0 +1,19 @@
+exports.handlePsqErrors = (err, req, res, next) => {
+    if (err.code === "22P02") {
+        res.status(400).send({ msg: "bad request"});
+    } else {
+        next(err);
+    }
+};
+
+exports.handleCustomErrors = (err, req, res, next) => {

```

Comment:
same as above here, if we don't have any error tests then we don't have any need to add these yet